fmralign/hyperalignment/local_template.py

```diff
@@ -0,0 +1,262 @@
+""" Local template computation functions. Those functions are part of the warp hyperalignment
+introducted by Feilong Ma et al. 2023. The functions are adapted from the original code and
+adapted for more general regionations approches.
```

Comment:
Not clear on the current phrasing, so offering one suggestion here : 

```suggestion
adapted for more general approches.
```

Otherwise could you confirm how this is generalized ?